### PR TITLE
issue-31-udp-cc

### PR DESCRIPTION
**Pull request by rinadina**

### CODE REVIEW

File: go_covert_lib/controller/channel/tcpNormal/tcpNormal_test.go [hunk 1]
```diff
@@ -9,26 +9,6 @@ import (
 	"time"
 )
 
-// These tests must be run in sudo mode
```
Comment (TaylorAbraham) [inline go_covert_lib/controller/channel/tcpNormal/tcpNormal_test.go:12]: Why are we deleting all of this?

File: go_covert_lib/test.html [hunk 2]
```diff
@@ -92,7 +92,7 @@
       }
 
       function selectConfig(configData, parent) {
-        console.log(event);
+        //console.log(event);
```
Comment (TaylorAbraham) [inline go_covert_lib/test.html]: We shouldn't leave commented out code in a merge

File: go_covert_lib/controller/channel/tcpNormal/tcpNormal_test.go (hunk 1 above)
Comment (rinadina) [reply go_covert_lib/controller/channel/tcpNormal/tcpNormal_test.go:12]: @MichaelDysart  said the comments are not applicable in this test

File: go_covert_lib/controller/channel/udpIP/udpIP_test.go [hunk 3]
```diff
@@ -0,0 +1,148 @@
+package udpIP
+
```
Comment (JacksonSchoe) [inline go_covert_lib/controller/channel/udpIP/udpIP_test.go:2]: Are commenting test classes at all?

**Review by JacksonSchoe — APPROVED** [review]

**Review by MichaelDysart — APPROVED** [review]

**Pull request merged**